pygerduty/v2.py

```diff
@@ -113,6 +114,12 @@ def ids_to_objs(key, value):
             new_kwargs.append(new_kwarg)
         return new_kwargs
 
+    def _apply_default_kwargs(self, kwargs):
+        for k, v in self.default_query_params.iteritems():
```

Comment:
`iteritems` is not py3 compatible. You'll have to either use `.items()` or `six.iteritems(...)`